website/react-magma-docs/src/components/MainNav/index.js

```diff
@@ -10,48 +10,106 @@ import {
   AccordionItemTitle,
   AccordionItemBody,
 } from 'react-accessible-accordion';
```

Comment:
Why are we using the `react-accessible-accordion` instead of our own Accordion component?

Reply:
Will create ticket to for this to update the existing version to our Accordion.